utils/docker-compose/Makefile

```
@@ -1,14 +1,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=compose
-PKG_VERSION:=2.26.0
+PKG_VERSION:=2.26.1
 PKG_RELEASE:=1
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

This does not look right to me because PKG_SOURCE should always (if possible, of course) contain PKG_NAME and PKG_VERSION at the same time.

```suggestion
PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
```

---

The reason, why I suggested and proposed it, it is because of dl folder within build system or you can find it here: https://sources.cdn.openwrt.org/

The file, you are downloading is gonna be called ``v2.26.1.tar.gz`` instead of ``docker-dompose.v2.26.1.tar.gz``, which could be overwritten by other packages and as well it is makes harder to know on the first sight with the missing name, what you downloaded. In the past, you can see that there were docker-compose files  with versioning.

---

I hadn't realized this, thanks for pointing it out.

---

Could we split it into two commits, please? I would like to backport the adding PKG_SOURCE to the stable branches as well. If not, thats okay as well, however in that case, add it to commit description as it otherwise looks like ordinary package bump with additional changes. It is just my point of view.